app/controllers/businesses_controller.rb

```diff
@@ -26,8 +26,30 @@ def update
     end
   end
 
+  def new
+    @business = Business.new
+  end
+
+  def create
+    user = current_user
+    @business = Business.new(new_business_params)
+    user.businesses << @business
```

Comment:
change to be more secure:
@business = current_user.businesses.create(new_business_params)